client/components/tinymce/plugins/mentions/plugin.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import tinymce from 'tinymce/tinymce';
+import ReactDom from 'react-dom';
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Mentions from './mentions';
+import sitesFactory from 'lib/sites-list';
+
+const sites = sitesFactory();
+
+function mentions( editor ) {
+	let node, siteId;
```

Comment:
why are we declaring these here? it doesn't look like they are used outside of the scope from `editor.on( … )`